### PR TITLE
Updated docs for SMP feature availability

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by abhijeet-harness — APPROVED** [review]

**Review by iamanmolm — CHANGES_REQUESTED** [review]
1. Kindly make "Handling Kubernetes secrets" part AWS, GCP similar to Azure:
Screenshot 2024-05-06 at 12 34 30 PM
Screenshot 2024-05-06 at 12 34 42 PM.

2. Also increase the architecture diagram size in both AWS, GCP as we did in Azure.

File: docs/cloud-cost-management/get-started/ccm-smp/1-smp-ccm-roadmap.md [hunk 1]
```diff
@@ -10,10 +10,35 @@ redirect_from:
...
-| Beta (Current version) | <ul><li>Visualize cost data from AWS commercial and GovCloud as well as Kubernetes clusters through perspectives and dashboards</li><li>Detect anomalies, set budgets and receive alerts, and generate Kubernetes workload and node pool recommendations</li><li>View cost data from Azure commercial and GovCloud</li><li>Support for Airgap on AWS</li><li>Fixing past 50 CVEs</li></ul> | <ul><li> AWS Commercial Cloud</li><li>Connected</li><li>Signed certificates</li><li> AWS Commercial Cloud</li><li>Connected/Airgapped</li><li>Signed certificates</li></ul>| <ul><li>AWS Commercial and GovCloud</li><li>Kubernetes clusters</li><li>AWS Commercial and GovCloud</li><li>Azure commercial and GovCloud</li><li>Kubernetes clusters</li></ul> | Istio virtual services for Azure in strict mode |
-| Limited GA (Coming soon)| In addition to the features mentioned above, the following features are supported: <ul><li>Autostopping</li><li>Support for Airgap on GCP and Azure</li></ul> | <ul></ul> | <ul></ul> | Istio virtual services for GCP and Azure in strict mode |
-| General Availability (Coming soon)| Feature parity with SaaS | 
+|Perspectives | ✅  | ❌  | ❌  | ✅ |
```
Comment (sdsingh07) [inline docs/cloud-cost-management/get-started/ccm-smp/1-smp-ccm-roadmap.md:35]: We are now supporting GCP and Azure as well.

File: docs/cloud-cost-management/shared/ccm-supported-platforms.md [hunk 2]
```diff
@@ -605,22 +605,40 @@ AWS is the only supported installation infrastructure. If you do not install Har
...
+|AutoStopping | ❌ | ❌ | ❌ | ❌ | 
+|Asset governance | ❌ | ❌ | ❌ | ❌ | 
+|Perspective Preferences| ✅ | ✅ | ✅ | ✅ |
+
+
+#### Air-gapped environment
+
+| **Features** | **AWS** | **Azure** | **GCP** | **Kubernetes** | 
+| --- | --- | --- | --- | --- |
+|Perspectives | ✅  | ❌  | ❌  | ✅ |
```
Comment (sdsingh07) [inline docs/cloud-cost-management/shared/ccm-supported-platforms.md:628]: We are now supporting GCP and Azure as well.

Comment (abhijeet-harness) [reply docs/cloud-cost-management/shared/ccm-supported-platforms.md:628]: This section is for supported feats. in AirGapped mode.

File: docs/cloud-cost-management/get-started/ccm-smp/1-smp-ccm-roadmap.md (hunk 1 above)
Comment (abhijeet-harness) [reply docs/cloud-cost-management/get-started/ccm-smp/1-smp-ccm-roadmap.md:35]: This section is for supported feats. in AirGapped mode.

**Pull request merged**